pkg/log/logger.go

```diff
@@ -178,13 +177,7 @@ func (l *gosoLogger) log(level int, msg string, args []interface{}, loggedErr er
 	timestamp := l.clock.Now()
 
 	for _, handler := range l.handlers {
-		if handler.Level() > level {
-			continue
-		}
-
-		handlerChannels := handler.Channels()
-
-		if !l.shouldLogToChannels(l.data.Channel, handlerChannels) {
+		if !l.shouldLog(l.data.Channel, level, handler) {
```

Comment:
seems like we only use it here, maybe we should instead have a function shouldNotLog

Reply:
ignoring, this is a common usage